backend/src/main/java/codezap/global/rds/DataSourceConfig.java

```diff
@@ -0,0 +1,63 @@
+package codezap.global.rds;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.sql.DataSource;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.jdbc.DataSourceBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.DependsOn;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.Profile;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.jdbc.datasource.LazyConnectionDataSourceProxy;
+
+import com.zaxxer.hikari.HikariDataSource;
+
+import lombok.RequiredArgsConstructor;
+
+@Configuration
+@RequiredArgsConstructor
+@Profile("prod")
+@EnableJpaRepositories(basePackages = "codezap")
+public class DataSourceConfig {
+
+    @Bean
+    @ConfigurationProperties("spring.datasource.write")
+    public DataSource writeDataSource() {
+        return DataSourceBuilder.create().type(HikariDataSource.class).build();
+    }
+
+    @Bean
+    @ConfigurationProperties("spring.datasource.read")
+    public DataSource readeDataSource() {
+        return DataSourceBuilder.create().type(HikariDataSource.class).build();
+    }
+
+    @Bean
+    @DependsOn({"writeDataSource", "readeDataSource"})
+    public DataSource routeDataSource() {
+        DataSourceRouter dataSourceRouter = new DataSourceRouter();
+        DataSource writerDataSource = writeDataSource();
+        DataSource readerDataSource = readeDataSource();
+
+        Map<Object, Object> dataSourceMap = new HashMap<>();
+        dataSourceMap.put("write", writerDataSource);
```

Comment:
```suggestion
        dataSourceMap.put(DataSourceRouter.WRITER_LOOKUP_KEY, writerDataSource);
```
위의 키들은 `DataSourceRouter` 의 반환값과 맞춰 주어야 하는건가요??
그렇다면 오탈자 때문에 오류가 나는 것을 예방하기 위해 상수로 관리하는 것은 어떤가요??

Reply:
좋은데요???!